backend/src/test/java/ai/verta/modeldb/DiffTest.java

```diff
@@ -107,6 +136,19 @@ public static void setServerAndService() throws Exception {
     }
   }
 
+  private final int blobType;
+  private final int commitType;
+
+  @Parameters
+  public static Collection<Object[]> data() {
```

Comment:
add a comment what these numbers indicate are for quick reference. Also if you can summarize what combinations are being tested either at the top of the file or at individual test , that will be super helpful

Reply:
fixed